setup/l10n_it_fatturapa/setup.py

```diff
@@ -2,5 +2,11 @@
 
 setuptools.setup(
     setup_requires=['setuptools-odoo'],
-    odoo_addon=True,
+    odoo_addon={
+        'external_dependencies_override': {
+            'python': {
+                'pyxb': 'PyXB==1.2.6',
```

Comment:
```suggestion
                'pyxb': 'PyXB==1.2.5',
```
Come già detto dagli altri, in v8 c'è `1.2.5`: https://github.com/OCA/l10n-italy/blob/be2711a9cac6ca33af6abeb33d4b90b78000784e/l10n_it_fatturapa/bindings/_ds.py#L5

Direi che questa PR si può mergiare una volta corretta la versione, non aspetterei l'arrivo di `1.2.6`.